android/src/main/java/io/wazo/callkeep/VoiceConnectionService.java

```diff
@@ -94,132 +139,81 @@ private Boolean canMakeOutgoingCall() {
     }
 
     private Connection createConnection(ConnectionRequest request) {
-        connection = new Connection() {
```

Comment:
This was moved and updated to VoiceConnection.java instead of being inline in the service.